pages/[measureId]/coverage.tsx

```diff
@@ -1,34 +1,46 @@
-import { Center, createStyles, Group } from '@mantine/core';
+import { Container, Group, ScrollArea, Tabs } from '@mantine/core';
+import parse from 'html-react-parser';
 import { useRouter } from 'next/router';
 import BackButton from '../../components/BackButton';
-import parse from 'html-react-parser';
-
-const useStyles = createStyles(() => ({
-  highlightedMarkup: {
-    '& pre': {
-      whiteSpace: 'pre-wrap'
-    }
-  }
-}));
+import classes from './coverage.module.css';
 
 const ClauseCoveragePage = () => {
-  const { classes } = useStyles();
   const router = useRouter();
-  const { clauseCoverageHTML, measureId } = router.query;
+  const { clauseCoverageHTML, clauseUncoverageHTML, measureId } = router.query;
 
-  if (typeof clauseCoverageHTML === 'string' && clauseCoverageHTML.length > 0) {
+  if (
+    typeof clauseCoverageHTML === 'string' &&
+    clauseCoverageHTML.length > 0 &&
+    typeof clauseUncoverageHTML === 'string' &&
+    clauseUncoverageHTML.length > 0
+  ) {
     return (
-      <Center>
-        <div>
-          <Group>
-            <BackButton />
-            <h2>Clause coverage for measure bundle: {`${measureId}`}</h2>
-          </Group>
-          <div className={classes.highlightedMarkup}>{parse(clauseCoverageHTML)}</div>
-        </div>
-      </Center>
+      <Container size={'90%'}>
+        <Group>
+          <BackButton />
+          <h2>Clause coverage for measure bundle: {`${measureId}`}</h2>
+        </Group>
+        <Tabs defaultValue="coverage" classNames={classes}>
+          <Tabs.List grow>
+            <Tabs.Tab value="coverage" /*leftSection={<IconPhoto style={iconStyle} />} */>Coverage</Tabs.Tab>
+            <Tabs.Tab value="uncoverage" /*leftSection={<IconMessageCircle style={iconStyle} />}*/>Uncoverage</Tabs.Tab>
```

Comment:
```suggestion
            <Tabs.Tab value="coverage">Coverage</Tabs.Tab>
            <Tabs.Tab value="uncoverage">Uncoverage</Tabs.Tab>
```
I think we can delete the commented out stuff